app/main.py

```diff
@@ -1,8 +1,30 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.__class__.people[name] = self
```

Comment:
In this case it is better to access people attrbibute through 'Person' class like so: Person.people["name"].
This makes access faster because if you use self.__class__.people[name] construct, python will first looks up the class  of the instance object (self), and then will access the people attribute in that class (that's an extra step). Otherwise it will address the class directly.